WooCommerce/WooCommerceUITests/Flows/LoginFlow.swift

```diff
@@ -0,0 +1,15 @@
+import UITestsFoundation
+import XCTest
+
+class LoginFlow {
+
+    // Login with WordPress.com account flow
+    @discardableResult
+    static func login() throws -> MyStoreScreen {
```

Comment:
Good job on extracting this into a separate flow class 👍 

One nitpick: WDYT about renaming it to something like this:

```suggestion
    static func logInWithWP() throws -> MyStoreScreen {
```

Which will make a comment not necessary then (also me being extra-nitpicky about logIn being a verb in this case, not a noun).